prototype/.babelrc

```diff
@@ -0,0 +1,3 @@
+{
+    "presets": ["env", "stage-2", "react"]
```

Comment:
Mentioned this via Slack but it turns out the order in which these presets are listed is significant, so I believe this needs to be `["env", "react", "stage-2"]` to avoid an error related to not being able to parse the class-properties method syntax.

Reply:
Did it work? I tried reordering those on an unrelated project with the same problem and it didn't help.

Reply:
Ooops, I lied: we may need to explicitly include `babel-plugin-transform-class-properties`. I'll push a commit with this.